### PR TITLE
Add Requester Name & Avatar to np embed

### LINKED ISSUE #360: Requester info in ;;nowplaying
Add the user name of the song requester and maybe their avatar(?) to the bottom of ;;np

### PR DESCRIPTION
**Pull request by Nanabell**
fixes #360

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/messaging/CentralMessaging.java [hunk 1]
```diff
@@ -454,6 +454,10 @@ public static EmbedBuilder addFooter(EmbedBuilder eb, Member author) {
         return eb.setFooter(author.getEffectiveName(), author.getUser().getAvatarUrl());
     }
 
+    public static EmbedBuilder addNpFooter(EmbedBuilder eb, Member requester) {
+        return eb.setFooter("Requested by" + requester.getEffectiveName(), requester.getUser().getAvatarUrl());
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/messaging/CentralMessaging.java]: Missing a whitespace after the "by".
Could you also please add `@Nonnull` / `@Nullable` annotations to result and parameters (whichever fits the intended method contract).

Comment (napstr) [reply FredBoat/src/main/java/fredboat/messaging/CentralMessaging.java]: Oh and please add a `//todo escape markdown` comment.

Comment (Nanabell) [reply FredBoat/src/main/java/fredboat/messaging/CentralMessaging.java]: I don't think you can have use markdown in a footer, can you?

Comment (napstr) [reply FredBoat/src/main/java/fredboat/messaging/CentralMessaging.java]: My bad, you're right. Should have tested it. Embeds are confusing with regards to where they allow markdown and where not.

**Pull request merged**